### PR TITLE
feat(Elife): Introduce article type

### PR DESCRIPTION
**Pull request by nlisgo**
The styles are close but will need a little attention from @discodavey on his return.

### CODE REVIEW

File: src/themes/elife/lib/query.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export interface articleData {
   pdf: string
   figuresPdf?: string
   copyright: { license: string }
+  type: 'replication-study' | 'research-article'
```
Comment (nlisgo) [inline src/themes/elife/lib/query.ts:10]: We need to confirm all of the article types for which we may have an ERA companion.

File: src/themes/elife/lib/articleType.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { create, prepend } from '../../../util'
+import { getType } from './dataProvider'
+import { articleType, articleData } from './query'
+
+const articleTypes = new Map([
+  ['research-article', 'Research Article'],
+  ['replication-study', 'Replication Study'],
```
Comment (nlisgo) [inline src/themes/elife/lib/articleType.ts:7]: I need to confirm all the types of article that may have an ERA companion article.

Comment (FAtherden-eLife) [reply src/themes/elife/lib/articleType.ts:7]: Here you go:
- Research Article
- Short Report
- Tools and Resources
- Research Advance
- Registered Report
- Replication Study
- Research Communication

Comment (nlisgo) [reply src/themes/elife/lib/articleType.ts:7]: Thanks @FAtherden-eLife, I have added all of these types. As new types are added would should extend this list.

**Comment by nlisgo** [comment]
@discodavey the test coverage has been resolved. Just need your attention on the css then I will mark it as ready for review.

**Pull request closed without merging**